src/Host/Client/Test/ExecutionTracing/BreakpointsTest.cs

```diff
@@ -18,6 +18,8 @@
 
 namespace Microsoft.R.ExecutionTracing.Test {
     [ExcludeFromCodeCoverage]
+    [Category.R.ExecutionTracing]
+    [Collection(CollectionNames.NonParallel)]
```

Comment:
It is important for these tests to run in parallel - that simulates multiple instances of RHost running concurrently